src/bottom-sheet.ts

```diff
@@ -2,10 +2,14 @@ import {postMessageToNativeApp, type SheetResponse} from './post-message';
 
 type SheetListType = 'SINGLE_SELECTION';
 
-type SheetListItem = {
+export type SheetListItem = {
```

Comment:
I miss here a deep readonly. Otherwise, this API will enforce to reveice a mutable object (and incorrectly meaning it may mutate the input objects, whereas it shouldn't)

Reply:
ok, changed